### DISCUSSION TITLE
Error running mininet on VirtualBox - Windows 11 with x86_64 on an M1 using UTM

### DISCUSSION THREAD

**Discussion by lgallez** [utmapp/UTM#4060] [General]
I was able to get Windows 11 with x86_64 up and running on my M1 using UTM. I'm attempting to use this setup for running [Mininet](http://mininet.org/download/) on [VirtualBox](https://www.virtualbox.org/wiki/Downloads). 
**Configuration**
* UTM Version: 3.1.5(53)
* OS Version: MacOS Monterey 12.2.1 2020 M1
*Apple Silicon

**Crash log**
[QEMULauncher-2022-05-21-115936.txt](https://github.com/utmapp/UTM/files/8748239/QEMULauncher-2022-05-21-115936.txt)

**Debug log**
[debug.log](https://github.com/utmapp/UTM/files/8748267/debug.log)

![Screen Shot 2022-05-21 at 10 48 56 AM](https://user-images.githubusercontent.com/40731237/169674297-cb4e79ad-c545-4a01-9902-58cdcf43ebee.png)
![Screen Shot 2022-05-21 at 10 48 29 AM](https://user-images.githubusercontent.com/40731237/169674308-61005cd1-e977-4bac-ae91-906e15068019.png)
![Screen Shot 2022-05-21 at 10 51 19 AM](https://user-images.githubusercontent.com/40731237/169666438-c751cff2-b363-44e1-8af0-f56b1915a560.png)

**Comment by brad-x** [comment]
It's actually really cool that you tried this, it shows how slick the VM experience in UTM is in certain ways.

VirtualBox won't succeed in running with UTM in this context because support for nested virtualization isn't present in several places (starting with macOS) - the good news is, mininet docs indicate you can run the VM in QEMU, which means you could install it directly with UTM - screenshot for effect

<img width="2560" alt="Screen Shot 2022-05-23 at 12 07 43 PM" src="https://user-images.githubusercontent.com/5598039/169862591-d2c53338-4df6-447a-bef1-08884087edca.png">

**Comment by lgallez** [comment]
Thank you for your comment!

I was kinda thinking this too after re-installing the spice-tools. I will def check out running mininet right off UTM.

**Comment by conath** [comment]
Moving to Discussions section as it seems more appropriate.

Regarding nested virtualization, the Apple Silicon M1-family chips do not support nested virtualization.
